src/deployment/AggregateDeployment.s.sol

```diff
@@ -88,6 +82,36 @@ contract AggregateDeployment is BaseDeployment {
             deploy.setUp();
             deploy.deployAndList();
         }
+
+        emit log("--- AAVE v2 ---");
+        {
```

Comment:
Should list the ERC4626 bridge twice, for 500K and 400K gas to have parity to mainnet.